vcl_templates/www.vcl.erb

```diff
@@ -250,7 +250,28 @@ sub vcl_recv {
       set req.http.GOVUK-ABTest-EducationNavigation = "A";
     }
   }
-  
+
+  <% ['SearchMatchLength'].each do |test_name| %>
+  if (req.http.User-Agent ~ "^GOV\.UK Crawler Worker") {
```

Comment:
Do you think we can skip the crawler logic? I don't think the crawler is going to see search results, so we can probably drop this (unless we think this logic is going to be shared by the performance platform, who might care about which version the crawler saves).